### ISSUE TITLE
Difference between 'no-data' and 'background' classes

### ISSUE THREAD

**Issue by wpkel** [Junjue-Wang/LoveDA#23]
Hi, I was wondering what is the difference between _no-data_ and _background_ classes in terms of their meaning (mentioned here: https://github.com/Junjue-Wang/LoveDA#dataset-and-contest), assuming that _background_ means land-cover classes that are not of interest? 

In terms of the class indices, I believe there might be an offset by 1 based on my own exploration and the class indices given here https://codalab.lisn.upsaclay.fr/competitions/421#learn_the_details-evaluation, resulting in _no-data_ class having index -1, _background_ 0, ..., _agriculture_ 6.

**Comment by Junjue-Wang** [comment]
Yes. The no-data region shows totally (0 ，0，0)  which often locates in the boundary of the image due to the geographical registration.
